transforms/index.js

```diff
@@ -52,10 +55,54 @@ exports.afterParse = function(parser) {
     
       // add the new function to the AST as an exported function
       entrySrc.statements.push(callWrapperStmt);
+    }
+
+  })
+
+  // this will need to be changed to support multiple capabilities
+  writeZomeJSON(zomeFuncs, '../capabilities/main/autogen.zome.json')
```

Comment:
@willemolding I was just thinking that one other possibility is writing directly to the manifest.json at the `zome` level, rather than creating these directories, with sub-json files. 

You know, I am thinking let's not do that yet, but I'll flag it as an idea that we can maybe follow up on in the future

Reply:
It would also be nice to not have this hard-coded in the function call so it is easy to change later if the app-spec changes